opencog/persist/sql/README.md

```diff
@@ -319,7 +319,7 @@ Each step discussed below.
 
 Database Install
 ----------------
-Download and install Postgres version 9.3 or newer.  The current design
+Download and install Postgres version 9.5.  The current design
```

Comment:
Hmm. Can you make it 9.5 or newer? It does work with 9.6, and the change in version 10 seems easy to fix.

Any clue about circleci? Its failing with `Permission denied (publickey).`

Reply:
Ok, added 'or newer'.

Regarding `circleci` I think it fails because I did create branch in the main repo not in the fork, but it is just hypothesis.

Reply:
Oh. Please don't create branches in the main repo...

Reply:
I mean, not for ordinary pull requests; create branches only for, aumm "major" things.

Reply:
Ah, OK. Not big a deal. I was afraid that this might be the start of a pull-req trend.